File: Pyvvo.Logistics/ClientApp/src/app/service/Model/shipping-method.ts
import { User } from "./user";
import { Status } from "./status";
import { Currency } from "./currency";
import { Parameter } from "./parameter";
import { CarrierService } from "./carrier-service";

export class ShippingMethod {
  id: number;
  createdOn: Date;
  createdOnLocalTime: string;
  updatedOn: Date;
  updateOnLocalTime: string;
  name: string;
  rate: number;
  status: Status;
  minParameter: Parameter;
  maxParameter: Parameter;
  currency: Currency;
  carrierService: CarrierService;
  isActive: boolean;
  createdBy: User;
}

File: Pyvvo.Logistics.Model/Model/ShippingMethod.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Pyvvo.Logistics.Model
{
    public class ShippingMethod
    {
        [Required, Key] public long Id { get; set; }
        [Required] public long StatusId { get; set; }
        [Required] public long MinParameterId { get; set; }
        [Required] public long MaxparameterId { get; set; }
        [Required] public long CurrencyId { get; set; }
        [Required] public long CreatedById { get; set; }
        [Required] public long CarrierServiceId { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public bool IsActive { get; set; }
        public string Name { get; set; }
        public double Rate { get; set; }
        [Required] public Status Status { get; set; }
        public Parameter MinParameter { get; set; }
        public Parameter MaxParameter { get; set; }
        public Currency Currency { get; set; }
        public User CreatedBy { get; set; }
        public CarrierService CarrierService { get; set; }

    }
}